playground/css-sourcemap/__tests__/css-sourcemap.spec.ts

```diff
@@ -29,7 +29,7 @@ describe.runIf(isServe)('serve', () => {
 
   test('linked css', async () => {
     const res = await page.request.get(
-      new URL('./linked.css', page.url()).href,
+      new URL('./linked-with-import.css', page.url()).href,
```

Comment:
We have a test for this case below. Can we test that the sourcemap doesn't exist?